drgn/helpers/linux/net.py

```diff
@@ -45,3 +43,25 @@ def sk_nulls_for_each(head: Object) -> Iterator[Object]:
         "struct sock", head, "__sk_common.skc_nulls_node"
     ):
         yield sk
+
+
+_NETDEV_HASHBITS = 8
+_NETDEV_HASHENTRIES = 1 << _NETDEV_HASHBITS
+
+
+def find_netdev_by_index(net: Object, ifindex: int) -> Object:
```

Comment:
The equivalent kernel function is called `dev_get_by_index()`, so we should either give it the same name or maybe name it `netdev_get_by_index()`.